tests/fixtures/api/issues/101/comments.json

```diff
@@ -0,0 +1,87 @@
+[
```

Comment:
I had to add all of these `comments.json` files because I'm not able to generate the old hash code because the `get_args` I'm receiving in my code is different from the old one:
https://github.com/webcompat/webcompat.com/blob/d2edc8ce4f303055e3ef7c82e64d730a4c4ca57e/webcompat/helpers.py#L393-L397
But if you think it's necessary I could generates new hashs.